app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -5,51 +5,57 @@ class AamvaStateMaintenanceWindow
     # All AAMVA maintenance windows are expressed in 'ET' (LG-14028),
     TZ = 'America/New_York'
 
+    Window = Data.define(:cron, :duration) do
+      def cron_val
+        Time.use_zone(TZ) { Fugit.parse_cron(cron) }
+      end
+    end.freeze
```

Comment:
another option, parse it in an initializer?
```suggestion
    Window = Data.define(:cron, :duration) do
      def initialize(:cron, :duration)
        super(
          cron: Time.use_zone(TZ) { Fugit.parse_cron(cron) },
          duration:
        )
      end
    end.freeze
```

Reply:
👏 This is the way.

I'm embarrassed to say that it somehow didn't occur to me that I could use an initializer here. But this feels like it solves the (minor) performance issue, and gets me away from bikeshedding names too.